hugo/content/upgrading.md

```diff
@@ -10,7 +10,38 @@ categories: ["configuration"]
 doctypes: ["task"]
 ---
 
-## Upgrade the NGINX Agent
+## Upgrade the NGINX Agent from version v2.31.0 or greater
+
+{{< important >}} In version v2.31.0 of the NGINX Agent, the behavior has changed so that the NGINX Agent will automatically restart itself during an upgrade. {{< /important >}}
```

Comment:
```suggestion
{{< important >}} Starting from version v2.31.0, NGINX Agent will automatically restart itself during an upgrade. {{< /important >}}
```

Here and going forward, we don't add "the" in front of NGINX products as per the style guide.